bus/pruning.go

```diff
@@ -0,0 +1,185 @@
+package bus
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	rhpv4 "go.sia.tech/core/rhp/v4"
+	"go.sia.tech/core/types"
+	cRHP4 "go.sia.tech/coreutils/rhp/v4"
```

Comment:
we should definitely follow up all our changes with a PR that makes this consistent across our codebase and then stick to our aliases

Reply:
https://github.com/SiaFoundation/renterd/issues/1678